<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<html><!-- GenHTML@12751-->
<head>
<META HTTP-EQUIV="Content-type" CONTENT="text/html; charset=iso-8859-1">
<!-- ident	"%Z%%M%	%I%	%E% SMI" -->

    <title>7.&nbsp;&nbsp;<tt>JTS</tt> Error Messages
    </title>
</head>
<body>
<font style="color: red; font-family:arial,helvetica,sans-serif; font-size:12pt; font-weight:bold}">Not For Publication</font>&nbsp;&nbsp;&nbsp;<br>Glassfish Review Draft<br> </td>

<A href="toc.html">Sun Java System Application Server Platform Edition 9.0 Error Message Reference</A>&nbsp;<br>
<A HREF="bfcvg.html">Previous</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<A HREF="toc.html">Contents</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<A HREF="bfcyf.html">Next</A><br>
            <!--startindex--><A NAME="bfcwu"></A><b>Chapter&nbsp;7</b><h1><tt>JTS</tt> Error Messages</h1>
<p>This chapter describes messages for errors with a <tt>JTS</tt> prefix.</p>
<A NAME="fyffx"></A><h2><tt>JTS</tt> Messages</h2>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5001</pre>
</td></table><br><p><tt>Servant is already active.</tt></p><p><tt>Servant is not active.</tt></p><blockquote><p><b>Solution: </b>This is an Unexpected Internal Error. Please contact
Sun with the complete error log message.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5005</pre>
</td></table><br><p><tt>Recovery or resync process got interrupted.</tt></p><blockquote><p><b>Solution: </b>See the server log for more details. This is an Unexpected
Internal Error. Please contact Sun with the complete error log message.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5008</pre>
</td></table><br><p><tt>Transaction log directory is not accessible.</tt></p><blockquote><p><b>Solution: </b>Make sure that the Transaction log directory (<tt>transaction-service.tx-log-dir</tt> in server configuration file) is proper and
has the read, write permissions for the user of the application server.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5015</pre>
</td></table><br><p><tt>ORB may not be running.</tt></p><blockquote><p><b>Solution: </b>Make sure that ORB is running. If ORB is running,
this is an Unexpected Internal Error. Please contact Sun with the complete error log
message.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5020</pre>
</td></table><br><p><tt>Configuration problem while giving the log path.</tt></p><blockquote><p><b>Solution: </b>Check the Transaction log path in server configuration
file (<tt>transaction-service.tx-log-dir</tt>).</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5028</pre>
</td></table><br><p><tt><tt>XAResource.recover</tt> has thrown an exception during recovery.</tt></p><blockquote><p><b>Solution: </b>See the exception Stack trace for more details.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5032</pre>
</td></table><br><p><tt>Server could not communicate with the resource manager with in the retry limit.</tt></p><blockquote><p><b>Solution: </b>Make sure that resource manager is up and running
or increase the retry limit (<tt>transaction-service.retry-timeout-in-seconds</tt> in the server configuration file).</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5040</pre>
</td></table><br><p><tt>ORB may not be running.</tt></p><blockquote><p><b>Solution: </b>See the server log for more details. This is an Unexpected
Internal Error. Please contact Sun with the complete error log message.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5041</pre>
</td></table><br><p><tt>Database driver or Resource Adapter has thrown <tt>XAException</tt> with
the error message The resource manager is doing work outside a global transaction</tt></p><blockquote><p><b>Solution: </b>Check if the application is using the same XA pool
outside the transactional context as well as in the transactional context. Some drivers
do not allow this scenario. Another scenario to verify is, two separate XA pools pointing
to the same backend with the same credentials and the second connection is obtained
with in the same transaction, without closing the first connection.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5064</pre>
</td></table><br><p><tt>Unexpected exception thrown from <tt>XAResource.end</tt>.</tt></p><blockquote><p><b>Solution: </b>See the server log for more details.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5065</pre>
</td></table><br><p><tt>Transaction Log directory path is not valid or proper permissions are not there.</tt></p><blockquote><p><b>Solution: </b>Make sure that transaction log directory is valid
and files in that directory have read write permissions.</p>
</blockquote>
<p>&nbsp;</p><table cellpadding="4" border="1" cols="1" width="100%"><td><pre>JTS5066</pre>
</td></table><br><p><tt>Possible cause is that transaction logs are corrupted.</tt></p><blockquote><p><b>Solution: </b>Please clean up the <tt>transaction-service.tx-log-dir/tx/*</tt> files and restart the server.</p>
</blockquote>

            <!--stopindex-->
<br>
<A HREF="bfcvg.html">Previous</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<A HREF="toc.html">Contents</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<A HREF="bfcyf.html">Next</A>
</body>
</html>
